### DISCUSSION TITLE
Requesting IntelliJ WebStorm Official Plugin

### DISCUSSION THREAD

**Discussion by saa-chin** [prisma/prisma#14583] [Q&A] [answered]
I use WebStorm for all my web projects. I see the VSCode plugin works great but looking forward to hearing about the WebStorm plugin.

**Comment by nurul3101** [comment, marked as answer]
Hey @SachinFSP 👋 

We have a Feature Request for Webstorm Plugin here: [#1788](https://github.com/prisma/prisma/issues/1788)
Please feel free to add a 👍 to the request so that we could prioritise it.

Here are the two unofficial plugins at the moment:

- https://plugins.jetbrains.com/plugin/14282-prisma-schema-language
- https://github.com/olance/intellij-prisma

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
